chicagorequests/__init__.py

```diff
@@ -178,4 +189,8 @@ def main(
             unit="day",
         ):
             for result in day:
+                if json_file:
+                    json.dump(result, json_file, ensure_ascii=False, indent=2)
                 click.echo(json.dumps(result))
```

Comment:
if we are writing to a file, let's not also write to stdout.

Reply:
Well, but why not? Still might want to see it as it comes in. I was trying to be as unobtrusive to existing process as possible.